package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Order_Tracking",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
Consider specifying a more appropriate license for your project. The current ISC license may not be suitable for all types of projects.